models/chirp.js

```diff
@@ -2,9 +2,41 @@ const mongoose = require('mongoose');
 
 //Schema Config
 var chirpSchema = new mongoose.Schema({
-    body: String,
-    //emoji: ?
-    //photo: ?
+    body: {
+        type: String,
+        minlength: 1,
+        maxlength: 280
+    },
+    created: {
```

Comment:
Can this be changed to be called createdDate, so we know that its a date value.

Reply:
Yeah definetely. More specific.